packages/back-end/src/services/organizations.ts

```diff
@@ -164,6 +165,28 @@ export function getConfidenceLevelsForOrg(context: ReqContext) {
   };
 }
 
+export function getAISettingsForOrg(
+  context: ReqContext,
+  includeKey: boolean = false
+): {
+  aiEnabled: boolean;
+  openAIAPIKey: string;
+  openAIDefaultModel: TiktokenModel;
+} {
+  const aiEnabled = IS_CLOUD
```

Comment:
Do we want it to be cloud only?  If they are providing their own `openAIAPIKey` I don't see why we should limit to cloud.